CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Fixes
+
+- Using SentryOptions.SuppressSegfaults, users can now block duplicate errors from native due to dotnet NullReferenceExceptions - Defaults to false ([#3998](https://github.com/getsentry/sentry-dotnet/pull/3998))
+- Native iOS events are now exposed to the dotnet layer for users to hook through SentryOptions.BeforeSend and SentryOptions.OnCrashedLastRun ([#2102](https://github.com/getsentry/sentry-dotnet/pull/3958))
```

Comment:
I take it this fixes an erroneous changelog entry from a previous PR?

It would be tidier to have that in it's own PR (it's not related to this change)... but as long as this gets merged fairly quickly, it probably doesn't matter too much.

Reply:
The changelog shifting on me during the last merge.  This fixes both and could be merged today